web/lib/clients/jawnTypes/public.ts

```diff
@@ -23,6 +27,9 @@ export interface paths {
   "/v1/request/query": {
     post: operations["GetRequests"];
   };
+  "/v1/request/queryClickhouse": {
```

Comment:
I try not to camelcaseURLS

instead please use `query-clickhouse`

<img width="893" alt="image" src="https://github.com/user-attachments/assets/5d638049-d3ae-45d4-93af-588d043eac3e">